tests/test_parser.py

```diff
@@ -10,7 +10,7 @@
     Units,
 )
 
-from oralb_ble.parser import OralBBluetoothDeviceData
+from src.oralb_ble.parser import OralBBluetoothDeviceData
```

Comment:
The src shouldn't be needed here since the package is expected to be installed

Reply:
Ah yes - didn't mean to, only meant to change that locally for a test. good catch